pkg/client/client_test.go

```diff
@@ -188,3 +188,19 @@ func TestHttpStatusCodes(t *testing.T) {
 		})
 	}
 }
+
+func TestAddEventsEndpointUrlTrailingSlashIsHandledCorrectly(t *testing.T) {
+	t.Setenv("SCALYR_SERVER", "https://app.scalyr.com/")
+	cfg, err := config.New(config.FromEnv())
+	assert.Nil(t, err)
+	sc, err := NewClient(cfg, nil, zap.Must(zap.NewDevelopment()))
+	require.Nil(t, err)
+	assert.Equal(t, sc.addEventsEndpointUrl, "https://app.scalyr.com/api/addEvents")
+
+	t.Setenv("SCALYR_SERVER", "https://app.scalyr.com/")
```

Comment:
what is the difference from first scenario? should one of them not have the trailing slash?

Reply:
Yep, one shouldn't have it. Will fix it. Thanks.

Reply:
Fixed in 064dbca057c7e7517d54f9b7a7f6e93742ea793e.